runelite-client/src/main/java/net/runelite/client/plugins/microbot/gabplugs/karambwans/GabulhasKarambwansConfig.java

```diff
@@ -0,0 +1,51 @@
+package net.runelite.client.plugins.microbot.gabplugs.karambwans;
+
+import net.runelite.client.config.Config;
+import net.runelite.client.config.ConfigGroup;
+import net.runelite.client.config.ConfigItem;
+import net.runelite.client.config.ConfigSection;
+
+@ConfigGroup("GabulhasKarambwans")
+public interface GabulhasKarambwansConfig extends Config {
+    @ConfigSection(
+            name = "Guide",
+            description = "Guide",
+            position = 1
+    )
+    String guideSection = "Guide";
+
+    @ConfigItem(
```

Comment:
Please use @ConfigInformation("") under the @ConfigGroup for your script description. Please give a small explanation what the script does and what the requirements are.

The @ConfigInformation tag supports html